### ISSUE TITLE
[Issue]: Create 6.1.x branch

### ISSUE THREAD

**Issue by torrance** [ROCm/HIPCC#151]
### Problem Description

As per the [build guide for NVIDIA](https://rocm.docs.amd.com/projects/HIP/en/latest/developer_guide/build.html) systems, I need to check out the `$ROCM_BRANCH` branch of hipcc.git. e.g.:

```
      export ROCM_BRANCH=rocm-6.1.x
      git clone -b "$ROCM_BRANCH" https://github.com/ROCm-Developer-Tools/hip.git
      git clone -b "$ROCM_BRANCH" https://github.com/ROCm-Developer-Tools/clr.git
      git clone -b "$ROCM_BRANCH" https://github.com/ROCm-Developer-Tools/HIPCC.git
      git clone -b "$ROCM_BRANCH" https://github.com/ROCm/hipother.git
```

Currently, however, whilst `rocm-6.1.x` exists in the other repos, I am forced to check out the `rocm-6.0.x` branch for HIPCC as this is the latest existing branch.

(Note also the documentation entirely fails to mention the `hipother.git` branch.)

Please either create a branch to allow consistency in building for NVIDIA or update the build instructions for NVIDIA systems to reflect the correct process.

### Operating System

n/a

### CPU

n/a

### GPU

AMD Instinct MI250X

### ROCm Version

ROCm 6.1.0

**Comment by lamb-j** [comment]
Good catch, we do need to update that documentation. I'll kick off that effort.

The new home for HIPCC is at https://github.com/ROCm/llvm-project under the amd/hipcc directory.

The rocm-6.1.x branch you're looking for exists there. Also, under the AMD tab (https://rocm.docs.amd.com/projects/HIP/en/latest/install/build.html), there is a note about hipother.git repository. That should probably be copied to the Nvidia tab too though.
